RELEASE-NOTES.md

```diff
@@ -232,9 +257,9 @@ following instructions for each nodes:
 
 Bitcask file format has changed between Riak 1.4 and 2.0. It supports
 implicit upgrade of bitcask data files, while downgrading is not
-supported. This is why downgrading requires a script to translate data
-files.
-
+supported. This is why automatic downgrading requires a script to
```

Comment:
Sorry that my comment was unclear.

The sentence I wanted to point was: 

> Bitcask file format has changed between Riak 1.4 and 2.0. It supports
implicit upgrade of bitcask data files, while downgrading is not supported.

At first glance, readers may think Riak 2.0 does not support downgrading to 1.4 (once bitcask was upgraded). I want to add "automatic" to the sentence.

> Bitcask file format has changed between Riak 1.4 and 2.0. It supports
implicit upgrade of bitcask data files, while __automatic__ downgrading is not supported.